util-images/network/netperfbenchmark/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM golang:1.15.2-alpine3.12 AS build-env
```

Comment:
Why do you need alpine?

Can't you simply use golang:1.15.2?

There are frequent security vulneraibilities in those more complex images.

Reply:
Will change it to use golang image.